drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -31,6 +31,11 @@ class Hmis::Hud::Client < Hmis::Hud::Base
   has_many :current_living_situations, through: :enrollments
   has_many :hmis_services, through: :enrollments # All services (HUD and Custom)
 
+  has_one :warehouse_client_source, class_name: 'GrdaWarehouse::WarehouseClient', foreign_key: :source_id, inverse_of: :source
+  has_many :warehouse_client_destination, class_name: 'GrdaWarehouse::WarehouseClient', foreign_key: :destination_id, inverse_of: :destination
+  has_one :destination_client, through: :warehouse_client_source, source: :destination, inverse_of: :source_clients
+  has_many :source_clients, through: :warehouse_client_destination, source: :source, inverse_of: :destination_client
```

Comment:
I think we might be confused later by the presence of `source_clients`, and we don't need it. The HMIS client _is_ the source client. This should be sufficient:

```suggestion
  has_one :warehouse_client_source, class_name: 'GrdaWarehouse::WarehouseClient', foreign_key: :source_id, inverse_of: :source
  has_one :destination_client, through: :warehouse_client_source, source: :destination
```